src/content/ui-api/gsctl/list-keypairs.md

```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: list keypairs
 title: "'gsctl list keypairs' command reference"
-description: "The 'gsctl list keypairs' lists key pairs for a cluster."
+description: "Dokcumentation of the 'gsctl list keypairs' command, which prints a list of all key pairs currently valid for accessing a workload cluster."
```

Comment:
```suggestion
description: "Documentation of the 'gsctl list keypairs' command, which prints a list of all key pairs currently valid for accessing a workload cluster."
```

Reply:
This has a different style than most other command descriptions. `delete nodepool` also has a different style.
Most commands follow the "The ___ command shows..." style